lib/compat/wordpress-5.9/class-wp-theme-json-5-9.php

```diff
@@ -825,42 +825,6 @@ protected function get_preset_classes( $setting_nodes, $origins ) {
 		return $preset_rules;
 	}
 
-	/**
-	 * Converts each styles section into a list of rulesets
-	 * to be appended to the stylesheet.
-	 * These rulesets contain all the css variables (custom variables and preset variables).
-	 *
-	 * See glossary at https://developer.mozilla.org/en-US/docs/Web/CSS/Syntax
-	 *
-	 * For each section this creates a new ruleset such as:
-	 *
-	 *     block-selector {
-	 *       --wp--preset--category--slug: value;
-	 *       --wp--custom--variable: value;
-	 *     }
-	 *
-	 * @param array $nodes Nodes with settings.
-	 * @param array $origins List of origins to process.
-	 * @return string The new stylesheet.
-	 */
-	protected function get_css_variables( $nodes, $origins ) {
```

Comment:
I mightn't get a chance to give this a proper review before wrapping up today, but just a quick question — does this function need to be removed from the `5.9` file? I thought the method in `6.1` should override it?

Reply:
Good questions! 

> does this function need to be removed from the 5.9 file? 
No it doesn't

> I thought the method in 6.1 should override it?
Yes!

I think it's me not really knowing the right protocol. 

Some [PRs](https://github.com/WordPress/gutenberg/commit/4fed55bc606232d33c3cdb7b42af57a324a807de#diff-3959e23010dbcf2f82be63ced3609c4c2611bac857f0a513e5fdafe73ccc9d00L1572) have removed methods, others [don't](https://github.com/WordPress/gutenberg/commit/cb1b5fd86de46894c5a15220c8a0dba88454931c#diff-f28aa79997745a4a51f917012b45a809c9b45d55134b55379a718f5c751dddf4R573) 🤔 

What's your take @noisysocks ?

Reply:
I don't really think we should ever be changing files in `wordpress-5.9` after WP 5.9 is released except for e.g. if we're preparing a WP 5.9.x release.

Remember the ultimate point of these files is to:

- Add functionality to the plugin that does not exist in Core using filters and whatnot.
- Do so in a way that makes it clear how that functionality should be added to Core when the time comes.

Updating files in `wordpress-5.9` doesn't help us with that second point because, when it comes time to preparing patches for the WP 6.1 release, one isn't able to look at the files in `wordpress-6.1` and see all of the code that needs to be added to Core.

How you currently have it in this PR looks right to me!

Reply:
Thanks for clarifying that point! 🙇

Reply:
Looks like it's not a problem anyway after https://github.com/WordPress/gutenberg/pull/41306